src/ui/dialogs/multiUserJoin.ts

```diff
@@ -42,6 +43,7 @@ class MultiUserJoinDialog {
 
       this.accountElement = dom.find('select[name="account"]');
       this.serverInputElement = dom.find('input[name="server"]');
+      this.serverInputList = dom.find('#jsxc-serverlist');
```

Comment:
Would be great if we could use as few ids as possible.

Reply:
the input ist linked via id to the datalist... how would change it?